optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -264,6 +274,21 @@ export const PreferentialTrials: FC<{ studyDetail: StudyDetail | null }> = ({
         >
           Which trial is the worst?
         </Typography>
+        <IconButton
+          disabled={latestHistoryId === null || undoHistoryId !== null}
+          onClick={() => {
+            if (latestHistoryId === null) {
+              return
+            }
+            setUndoHistoryId(latestHistoryId)
+            action.removePreferentialHistory(studyDetail.id, latestHistoryId)
+          }}
+          sx={{
+            margin: "auto 0 auto auto",
+          }}
+        >
+          <UndoIcon />
+        </IconButton>
```

Comment:
My first impression is like this button is small and cannot understand what will be happened when clicked.

<img width="1552" alt="Screenshot 2023-09-15 18 37 33" src="https://github.com/optuna/optuna-dashboard/assets/5564044/899c7773-1a16-4fb6-ae44-892d64905b75">

How about adding a label "Undo" to this button like the top page?

<img width="1560" alt="Screenshot 2023-09-15 18 37 25" src="https://github.com/optuna/optuna-dashboard/assets/5564044/ea430812-2fe3-4a16-b67e-5127a0bba142">